rebar.config

```diff
@@ -58,10 +58,13 @@
                    {ref, "17d7a8d"}}},
 
         {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                     {ref,"720510a"}}},
+                     {ref,"34ae94e"}}},
 
         {aesophia, {git, "https://github.com/aeternity/aesophia.git",
-                     {ref,"4d9d307"}}},
+                     {ref,"36395b5"}}},
```

Comment:
It looks to me that we shall put a comment in this line reminding developer that the version dictated by this software may end up in the bytecode/transaction so for the sake of determinism reviewer shall be alerted that any time this line is changed then he/she shall run extra check on version having been adhered to. Not in scope of this PR though.

Reply:
Not a problem, this compiler is _NOT_ used to compile :-)

Reply:
The compiler used by the (deprecated) HTTP API is the checked in escript...